src/core/Block/__snapshots__/Block.test.tsx.snap

```diff
@@ -43,51 +43,131 @@ exports[`calling render with the same component on the same container does not r
 }
 
 .c0.fi-block--margin-xxs {
-  margin: 2px;
+  margin: 5px;
```

Comment:
These still require investigation. Not sure if this test has any real implications, or is it just used to test all combinations for spacing.

Reply:
So it looks like some props are generated out of tokens and now include the updated values and new inset-tokens as well. We should probably consider what to do with these.

Reply:
It was decided to exclude the inset tokens from the components public API's. Inset spacing is available in design tokens and can be used with traditional generic API's like style that is mostly supported by the components as well.